charts/agh2/README.md

```diff
@@ -81,4 +81,25 @@ imageCredential:
   registry: registry.lkc-lab.com
   username: robot$leukocyte-lab+username
   password: password123456789000
-```
\ No newline at end of file
+```
+---
+use above file as deployment.yaml    
+  
+If first time install agh, Need to add repo
+``` bash
+helm repo add lkclab https://charts.lkc-lab.com/
+```
+Install agh
+``` bash
+helm install agh lkclab/agh2-helm -f deployment.yaml --namespace agh --create-namespace
+```
+If want to up date,Need to update repo
+``` bash
+helm repo update
+```
+Update agh to new version
+``` bash
+helm upgrade agh2 lkclab/agh2-helm -f deployment.yaml --namespace agh
+```
+
+
```

Comment:
Unexpected double trailing blank lines.
Apply / Install linter or IDE extensions to improve in writing Markdown files, ex.: [:link: `markdownlint`](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint).